push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/NotificationDispatcher.java

```diff
@@ -50,27 +49,41 @@
     @Inject
     private PushMessageMetricsService pushMessageMetricsService;
 
+    private final String ADM_TOPIC = "agpush_AdmTokenTopic";
+
+    private final String APNS_TOPIC = "agpush_APNsTokenTopic";
+
+    private final String FCM_TOPIC = "agpush_FCMTokenTopic";
+
+    private final String MPNS_TOPIC = "agpush_MPNSTokenTopic";
+
+    private final String MOZ_TOPIC = "agpush_SimplePushTokenTopic";
+
+    private final String WNS_TOPIC = "agpush_WNSTokenTopic";
+
     /**
-     * Receives a {@link UnifiedPushMessage} and list of device tokens that the message should be sent to, selects appropriate sender implementation that
-     * the push notifications are submitted to.
+     * Consumes a record of type {@link UnifiedPushMessage} from {@link org.jboss.aerogear.unifiedpush.kafka.MessageHolderWithTokensKafkaProducer}
+     * with a list of device tokens that the message should be sent to, and selects the appropriate sender implementation that the push notifications
+     * are submitted to.
      *
      * @param msg object containing details about the payload and the related device tokens
      */
-    public void sendMessagesToPushNetwork(@Observes @Dequeue MessageHolderWithTokens msg) {
+    @Consumer(topics = {ADM_TOPIC, APNS_TOPIC, FCM_TOPIC, MPNS_TOPIC, MOZ_TOPIC, WNS_TOPIC}, groupId = "NotificationDispatcherKafkaConsumer_group")
```

Comment:
Can we have a group id as a static property. An example of such group id already used is "agpush_installationMetricsGroup". So maybe it would be good to keep them consistent.